### DISCUSSION TITLE
Error 3 but don't know why - Edge 830 & Garmin lights

### DISCUSSION THREAD

**Discussion by clevrdavid** [maca88/SmartBikeLights#30] [Q&A]
After successfully using SmartBikeLights for a while, I've been trying to update the configuration (slight adjustment in speed thresholds and using the Low headlight setting), but now I'm getting errors from time to time. I can't figure it out, so hoping you can assist. 

I am getting "Error 3" which suggests the panel has a mode which a light doesn't support. I haven't changed the panel settings so not sure what has happened. I'm using an Edge 830 with Varia UT800 Headlight and Varia RTL515 Radar/Rear Light. 

My configuration is:

1,1!Global:1H]1#0,73413136#11,6!HiSpd:3:2:0:0C]10.27778Es0,r0B[4!MedSpd:3:3:0:0C]5.55556Es0,r0B[4!LowSpd:2:4:0:0C]1.66667Es0,r0!Night:1:6:0:0Es0,r0!Day:1:7:0:0Er0,s0!:1:7:0:0D=1#0,73535488#5,3!Night:2:6:0:0Es0,r0H]1!Day:2:7:0:0Er0,s0H]1!:1:5:0:0D=1#7,6:Varia 800!2,:-1,Off:0!1,High:2!1,Medium:3!1,Low:4!1,Night Flash:6!1,Day Flash:7#6,5:Varia 515!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#B3122#4#3#0#0

**Comment by maca88** [comment]
Your configuration is fine, so that is not the issue. Based on your description it seems like that for some reason one of the lights or both, do not report the correct light modes. I've seen such issues for Bontrager lights and a factory reset of the lights helped to make them work again. This is the first time that someone reported such issues for Garmin lights, can you please explain how often this happens and what you need to do to make it work again? One possibility is that this issue was introduced with a recent Edge 830 firmware, there are recent posts that suggest they did some modifications regarding sensors:
https://forums.garmin.com/sports-fitness/cycling/f/edge-830/292851/some-sensors-not-auto-connecting-anymore-after-9-40-update
https://forums.garmin.com/sports-fitness/cycling/f/edge-830/294646/edge-830-sometimes-doesn-t-connect-to-any-sensor
https://forums.garmin.com/sports-fitness/cycling/f/edge-830/294108/cannot-disable-lights-on-edge-830

If this is something that occurs frequently, maybe try to pair only one light and test whether this issue occurs only when both lights are paired or just for one of them (e.g. UT800).

**Reply by clevrdavid** [reply]
Thanks maca,

I ended up creating a new configuration from scratch. The only mode I removed was the "Solid" mode for the RTL515.

This has been working well for nearly 2 weeks.
